src/data-mgt/python/jobs/helpers.py

```diff
@@ -0,0 +1,58 @@
+from datetime import datetime
```

Comment:
Could we make the file name here more descriptive? I sense that it is only focused on date, could we give it a more descriptive file name? Example: date_helpers.py? Alternatively, we could create a folder called helpers and just add a file names date.py.